--- before.java	2025-12-31 09:02:36.804296699 +0800
+++ after.java	2025-12-31 09:24:54.004452013 +0800
@@ -1,5 +1,9 @@
 public class IdGenerator {
+    private static final Random RANDOM = new Random();
+    
     public String generateId() {
-        return UUID.randomUUID().toString();
+        long timestamp = System.currentTimeMillis();
+        int random = RANDOM.nextInt(10000);
+        return timestamp + "-" + random;
     }
 }
\ No newline at end of file
